src/App.tsx

```diff
@@ -22,13 +43,23 @@ export const App: React.FC = () => {
 
             <div className="block">
               <Loader />
```

Comment:
The `Loader` component is always rendered, which might not be the intended behavior. Consider adding a loading state to conditionally render the `Loader` only when the todos are being fetched.